### PR TITLE
Implement to support tosa.depthwise_conv2d operation

### PR DESCRIPTION
**Pull request by inwchun**
implement for depthwise op

### CODE REVIEW

File: src/encode.cpp [hunk 1]
```diff
@@ -1035,7 +1136,7 @@ void encodeOp(State &st, mlir::tosa::Conv2DOp op, bool) {
 
   auto output = Tensor::mkInitializedLambda(
                   elemTy, move(outDims), move(outInd), 
-                  biasf.add(tf)
+                  tf.add(biasf)
```
Comment (aqjune) [inline src/encode.cpp:1139]: Why is this changed?

**Comment by aqjune** [comment]
You'll need to update `analyzeBlock` in `analysis.cpp` as well.

**Comment by aqjune** [comment]
Can we merge common parts of this and `encodeOp(State &st, mlir::tosa::Conv2DOp op, ..)`?

File: src/encode.cpp (hunk 1 above)
Comment (inwchun) [reply src/encode.cpp:1139]: to be same as the tosa spec

**Comment by aqjune** [comment]
Build is broken?

**Comment by aqjune** [comment]
Oops, the failures are from Z3 API updates. I will update them.

**Comment by aqjune** [comment]
Ehh, no, they are compile failures in the Z3 header.

File: src/encode.cpp (hunk 1 above)
Comment (aqjune) [reply src/encode.cpp]: Okay, fine.
FYI: since fp add is commutative, they are equivalent.

**Comment by inwchun** [comment]
I think it's an internal error in z3
